dnf-plugins-core.spec

```diff
@@ -26,7 +35,8 @@ protected_packages passive plugins.
 %package -n python-dnf-plugins-core
 Summary:    Core Plugins for DNF
 Group:      System Environment/Base
-BuildRequires:  python-dnf >= %{dnf_version}
+BuildRequires:  python-dnf >= %{dnf_lowest_compatible}
+BuildRequires:  python-dnf < %{dnf_not_compatible}
```

Comment:
This feels very strange to me, I what situation is this needed
I understand the need for a minimun dnf version, but why a maximum version